src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx

```diff
@@ -72,7 +72,7 @@ function CypherScriptFrame({
                       data-testid="multi-statement-list-title"
                       {...titleProps}
                     >
-                      <PointerFrameCommand selectedDb={frames[id].cmd}>
+                      <PointerFrameCommand selectedDb={frames[id].useDb || ''}>
```

Comment:
Nice find, is the precedent to use empty string vs null/undefined when there is no db?

Reply:
Fixed this in another commit